test/wasm2js/br_table_temp.2asm.js

```diff
@@ -12768,13 +12768,6 @@ function asmFunc(imports) {
   return $1_1 | 0;
  }
  
- function f($0_1, $1_1, $2_1) {
```

Comment:
Is this change for something else?

Reply:
No, this is an optimization unlocked by this PR. It is defined here:

https://github.com/WebAssembly/binaryen/blob/76ab43f42e47692c461d9cbe61864d76611cbc38/test/wasm2js/br_table_temp.wast#L971

It has a few direct calls, but they get optimized out in this optimized test output. Previously, I guess it remained alive because of this table reference, when we can now see has no corresponding call_indirect,

https://github.com/WebAssembly/binaryen/blob/76ab43f42e47692c461d9cbe61864d76611cbc38/test/wasm2js/br_table_temp.wast#L995